Objects/codeobject.c

```diff
@@ -640,7 +640,7 @@ PyCode_New(int argcount, int kwonlyargcount,
 
 static const char assert0[6] = {
     RESUME, 0,
-    LOAD_ASSERTION_ERROR, 0,
+    LOAD_EXCEPTION_TYPE, 0,  // AssertionError
```

Comment:
The mapping between the oparg and the const can be in opcode.py, and then used to generate #defines in opcode.h.

Reply:
Hm, maybe. I definitely think this would make more sense for `LOAD_COMMON_CONST` later, but it feels pretty heavy for just 2-3 exception class types. This is far from our most opaque oparg. ;)